pyanaconda/users.py

```diff
@@ -235,8 +271,173 @@ def guess_username(fullname):
     username = strip_accents(username).encode("utf-8")
     return username
 
+
+class PasswordCheckRequest(object):
+    """A wrapper for a password check request.
+
+    This in general means the password to be checked as well as its validation criteria
+    such as minimum length, if it can be empty, etc.
+    """
+
+    def __init__(self, password,
+                 username='root',
+                 minimum_length=None,
+                 empty_ok=False,
+                 pwquality_settings=None):
+        self._password = password
+        self._username = username
+        if minimum_length is None:
+            minimum_length = PASSWORD_MIN_LEN
```

Comment:
Move this few lines above. It will be more readable when this logic won't be inside of other assignments.